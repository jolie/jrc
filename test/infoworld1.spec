Post >= 2 and
CommentsServices >= 2 and
Gateway >= 1
