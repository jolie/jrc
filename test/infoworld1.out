{
 "configuration": [
  {
   "DC_number": 3, 
   "dependencies": [
    {
     "Auth": "SERVICE___U6CLB"
    }, 
    {
     "CommentsBalancer": "SERVICE___409BS"
    }
   ], 
   "id": "SERVICE___QQCNN", 
   "service": "CommentsServices", 
   "DC": "small_DC"
  }, 
  {
   "DC_number": 3, 
   "dependencies": [
    {
     "Auth": "SERVICE___U6CLB"
    }, 
    {
     "CommentsBalancer": "SERVICE___409BS"
    }
   ], 
   "id": "SERVICE___WP9PI", 
   "service": "CommentsServices", 
   "DC": "small_DC"
  }, 
  {
   "DC_number": 3, 
   "id": "SERVICE___U6CLB", 
   "service": "Auth", 
   "DC": "big_DC"
  }, 
  {
   "DC_number": 3, 
   "dependencies": [
    {
     "Auth": "SERVICE___U6CLB"
    }
   ], 
   "id": "SERVICE___5O4FD", 
   "service": "Post", 
   "DC": "big_DC"
  }, 
  {
   "DC_number": 3, 
   "dependencies": [
    {
     "Auth": "SERVICE___U6CLB"
    }
   ], 
   "id": "SERVICE___C0DAH", 
   "service": "Post", 
   "DC": "big_DC"
  }, 
  {
   "DC_number": 2, 
   "dependencies": [
    {
     "Auth": "SERVICE___U6CLB"
    }
   ], 
   "id": "SERVICE___QXWTL", 
   "service": "Gateway", 
   "DC": "big_DC"
  }, 
  {
   "DC_number": 1, 
   "id": "SERVICE___409BS", 
   "service": "CommentsBalancer", 
   "DC": "big_DC"
  }
 ]
}
